content/patterns/disclosure/examples/disclosure-navigation-hybrid.html

```diff
@@ -18,185 +18,185 @@
     <script src="js/disclosureMenu.js"></script>
   </head>
   <body>
-    <nav aria-label="Related Links" class="feedback">
+    <nav aria-label="関連するリンク" class="feedback">
       <ul>
-        <li><a href="https://github.com/w3c/aria-practices/projects/14">Related Issues</a></li>
-        <li><a href="../disclosure-pattern.html">Design Pattern</a></li>
+        <li><a href="https://github.com/w3c/aria-practices/projects/14">関連する Issues</a></li>
+        <li><a href="../disclosure-pattern.html">デザインパターン</a></li>
       </ul>
     </nav>
     <main>
-      <h1>Example Disclosure Navigation Menu with Top-Level Links</h1>
+      <h1>トップレベルリンク付きディスクロージャーナビゲーションメニューの例</h1>
+      <!-- af8f4c1d97ead402022d704754c3d382c4a2cef2 -->
 
       <section>
-        <h2>About This Example</h2>
+        <h2>この例について</h2>
         <div class="advisement">
-          <h3>Important</h3>
+          <h3>重要</h3>
           <p>
-            Although this example uses the word "menu" in the colloquial sense to refer to a set of navigation links, it does not use the WAI-ARIA menu role.
-            This implementation of site navigation does not use the menu role because it does not provide the complex functionality that assistive technologies expect in a widget that has the menu role.
-            Typical site navigation does not need all the
-            <a href="../../menubar/menu-and-menubar-pattern.html#keyboard_interaction">keyboard interactions specified by the menu and menubar pattern</a>.
+            この例では、口語的な意味で「メニュー」という言葉を使用してナビゲーションリンクのセットを参照していますが、WAI-ARIA のメニューロールは使用していません。
+            このサイトナビゲーションの実装では、支援技術がメニューロールを持つウィジェットに期待する複雑な機能を提供しないため、メニューロールは使用しません。
+            典型的なサイトナビゲーションには、<a href="../../menubar/menu-and-menubar-pattern.html#keyboard_interaction">メニュー及びメニューバーのパターンで指定されているすべてのキーボードインタラクション</a>は必要ありません。
```

Comment:
all the がリンクテキストに混じってしまった
```suggestion
            典型的なサイトナビゲーションには、<a href="../../menubar/menu-and-menubar-pattern.html#keyboard_interaction">メニュー及びメニューバーのパターンで指定されているキーボードインタラクション</a>のすべては必要ありません。
```